objectivec/README.md

```diff
@@ -133,6 +133,21 @@ be collisions. This option provides a prefix that will be added to the Enums
 and Objects (for messages) generated from the proto. Convention is to base
 the prefix on the package the proto is in.
 
+**GPB_OBJC_EXPECTED_PACKAGE_PREFIXES** (no default)
+
+This environment variable can be set when invoking protoc to point to a file
+that lists expected `package` to `objc_class_prefix` values.  The compiler will
+then issue warnings and/or errors if the data in the proto files being generated
+is not listed in the expectations file. The format for the file is:
+
+- An entry is done via `package=prefix`.
```

Comment:
Can we use something like protobuf text format instead? This avoid creating a parser for the configurations.

Also before this change, generators are deterministic solely on the parsed descriptor protos. This change will add additional input to the protoc. I'd rather explore other options before committing to this approach.